data/json/monsters/mammal.json

```
@@ -162,6 +162,38 @@
     "special_attacks": [ [ "EAT_FOOD", 120 ] ],
     "flags": [ "SEES", "SMELLS", "HEARS", "WARM", "SWIMS", "ANIMAL", "STUMBLES", "PATH_AVOID_DANGER_1" ]
   },
+  {
+    "id": "mon_lab_rat",
+    "type": "MONSTER",
+    "name": { "str": "lab rat" },
+    "description": "The albino Norway rat, an omnivorous rodent with sleek white fur and a long, rough tail.  While its ancestors were harbingers of plague, these clever animals are omnipresent in modern biomedical research.",
+    "default_faction": "rat",
+    "bodytype": "pig",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "320 ml",
+    "weight": "320 g",
+    "hp": 6,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "r",
+    "color": "white",
+    "aggression": -15,
+    "morale": -50,
+    "melee_skill": 3,
+    "melee_dice": 1,
+    "melee_dice_sides": 1,
+    "melee_cut": 1,
+    "dodge": 2,
+    "vision_day": 30,
+    "vision_night": 15,
+    "harvest": "mammal_tiny",
+    "path_settings": { "max_dist": 10 },
+    "fear_triggers": [ "PLAYER_CLOSE", "SOUND", "FRIEND_DIED", "FIRE" ],
```

```suggestion
    "fear_triggers": [ "PLAYER_CLOSE", "SOUND" ],
```
I forgot - FIRE and FRIEND_DIED are in their species entry ("MAMMAL") as fear triggers so are already there. @Venera3, what are your thoughts regarding, first, this one and, second, whether black rats should actually have STAGGER (and KEENNOSE, for that matter)?

---

Both `SMELLS` and `KEENNOSE` will make them actively path towards you, which might feel a bit weird for otherwise-passive critters.